tools/amd_build/patches/a_aten_src_THC_THCAllocator.c.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/aten/src/THC/THCAllocator.c b/aten/src/THC/THCAllocator.c
+index 554a379d7..4ff8b0c55 100644
+--- a/aten/src/THC/THCAllocator.c
++++ b/aten/src/THC/THCAllocator.c
+@@ -64,7 +64,7 @@ static void *THCUVAAllocator_alloc(void* ctx, ptrdiff_t size) {
+   // See J.1.1 of the CUDA_C_Programming_Guide.pdf for UVA and coherence rules
+   // on various compute capabilities.
+   void* ptr;
+-  THCudaCheck(cudaMallocManaged(&ptr, size, cudaMemAttachGlobal));
++  THCudaCheck(cudaSuccess);
```

Comment:
This should also be replaced with `throw ...` instead of silently saying everything went file